cms/djangoapps/contentstore/views/credit_eligibility.py

```diff
@@ -0,0 +1,3 @@
+"""
```

Comment:
No django doesn't require it. Actually there was a basic view to test it but I remove that.
I will delete this file.